portafly/doc/ux_specs/backend/create_backend.md

```diff
@@ -0,0 +1,44 @@
+# Create Backend
+
+[Back to index](../index.md)
+
+* [Mockups](https://marvelapp.com/prototype/12844cg4)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/APPDUX-343)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-5575)
+
+##### Title
+* Create a new backend
```

Comment:
@deernano We don't need to specify the title in the UX specs. The title will be specified in the UI strings.